Packs/AzureStorage/Integrations/AzureStorage/README.md

```diff
@@ -0,0 +1,637 @@
+Deploy and manage storage accounts and blob services.
+This integration was integrated and tested with version 2019-06-01 of Azure Storage
+## Permissions
+
+## Configure Azure Storage on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Azure Storage.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Application ID | True |
+    | Subscription ID | True |
+    | Resource Group Name | True |
+    | Trust any certificate (not secure) | False |
+    | Use system proxy settings | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
```

Comment:
Please add the text from the description about how the authentication works to the readme before the commands section.

Reply:
done.

Reply:
I added it at the start instead the permissions